Code/Editor/Plugins/ComponentEntityEditorPlugin/SandboxIntegration.cpp

```diff
@@ -465,6 +465,16 @@ AZ::Vector3 SandboxIntegrationManager::GetWorldPositionAtViewportCenter()
     return AZ::Vector3::CreateZero();
 }
 
+AZ::Vector3 SandboxIntegrationManager::GetWorldPositionAtViewportInteraction() const
+{
+    if (GetIEditor() && GetIEditor()->GetViewManager())
+    {
+        return GetIEditor()->GetViewManager()->GetClickPositionInViewportSpace();
+    }
+
+    return AZ::Vector3::CreateZero();
```

Comment:
nit: Reduce duplicate `Get` calls
```suggestion
    auto* editor = GetIEditor();
    auto* viewManager = (editor != nullptr) ? editor->GetViewManager() : nullptr;
    return(viewManager != nullptr) ? viewManager->GetClickPositionInViewportSpace() : AZ::Vector3::CreateZero();
```

Reply:
I refactored it a little bit, but addressed the multiple `Get` calls.

```
AZ::Vector3 SandboxIntegrationManager::GetWorldPositionAtViewportInteraction() const
{
    const auto& iEditor = GetIEditor();
    if (const auto& viewManager = (iEditor != nullptr) ? iEditor->GetViewManager() : nullptr)
    {
        return viewManager->GetClickPositionInViewportSpace();
    }

    return AZ::Vector3::CreateZero();
}
```